README.md

```diff
@@ -2,21 +2,36 @@
 
 # cardano-benchmarking
 
-## preparation
+This repository contains two important tools for `cardano-node` benchmarking:
+
+1. `cardano-tx-generator`
+2. `cardano-rt-view-service`
```

Comment:
```suggestion
2. [cardano-rt-view-service](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-rt-view)
```

Reply:
But I already provided a link to the project below.

Reply:
Done.